doc/source/security/encrypt-traffic.rst

```diff
@@ -12,19 +12,84 @@ This includes traffic between client applications and cluster nodes,
    :local:
    :depth: 1
 
+There are two approaches to configure encryption: automatic and manual.
+
+.. _ssl-auto-conf:
+
+SSL Automatic Configuration
+===========================
+
+Automatic configuration needs two steps to be done:
+
+1. :ref:`creating-certs`
+2. :ref:`enabling_encrypt-cluster-traffic`
+
+.. _creating-certs:
+
+Creating Certificates
+---------------------
+
+Automatic configuration of the SSL encryption needs key and certificate files.
+These files are auto-generated by |MySQL| by default, and no manual operations
+are needed to use this default variant. 
+
```

Comment:
If we use auto-generate certificates from mysql, pxc node will terminate with following error
```
2018-03-22T05:33:30.274087Z 0 [Note] WSREP: gcomm: connecting to group 'cluster_one', peer '192.168.100.10:,192.168.100.20:,192.168.100.30:'
2018-03-22T05:33:30.276357Z 0 [ERROR] WSREP: handshake with remote endpoint ssl://192.168.100.20:4567 failed: asio.ssl:67567722: 'block type is not 01' ( 67567722: 'error:0407006A:rsa routines:RSA_padding_check_PKCS1_type_1:block type is not 01')
2018-03-22T05:33:30.276576Z 0 [ERROR] WSREP: handshake with remote endpoint ssl://192.168.100.10:4567 failed: asio.ssl:67567722: 'block type is not 01' ( 67567722: 'error:0407006A:rsa routines:RSA_padding_check_PKCS1_type_1:block type is not 01')
2018-03-22T05:33:30.279618Z 0 [Note] WSREP: SSL handshake successful, remote endpoint ssl://192.168.100.30:33532 local endpoint ssl://192.168.100.30:4567 cipher: AES128-SHA compression: none
2018-03-22T05:33:30.279748Z 0 [Note] WSREP: SSL handshake successful, remote endpoint ssl://192.168.100.30:4567 local endpoint ssl://192.168.100.30:33532 cipher: AES128-SHA compression: none
2018-03-22T05:33:30.279943Z 0 [Note] WSREP: (8dd3a4df, 'ssl://0.0.0.0:4567') connection established to 8dd3a4df ssl://192.168.100.30:4567
```
Please add one note as;
```
Note

You must use the same key and certificate files on all nodes.
```